pyvalidator/is_freight_container.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+def is_valid_container_number(container_number)-> bool:
```

Comment:
Make sure to match the name of the function with the file. This is the convention that the project is following.